### DISCUSSION TITLE
Can't open files from server

### DISCUSSION THREAD

**Discussion by JorgenM** [phcode-dev/phoenix#1771] [General]
Hi, I'm a happy Brackets-user, but was hinted to Phoenix today when I started Brackets. I have problems open files directly from the web-server - is there a setting I should change to be able to that?

**Comment by abose** [comment]
Phoenix cannot open network drives in windows yet, we are in the process of migrating most features from Brackets. This should be fixed in the next 2/3 releases. Phoenix code usually has a monthly release cycle, so this should come by October.

* https://github.com/phcode-dev/phoenix/issues/1757

**Comment by JorgenM** [comment]
Okay, I will check Phoenix out again later this year - keep up the good work!
